actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -2,6 +2,25 @@
 
 module ActionDispatch
   module SystemTesting
+    # Deletes screenshots older than the provided TTL (time to live) in seconds.
+    #
+    # This method is supposed to be called in the beginning of a test suite run and after
+    # test files and configuration have been loaded.
+    def self.rotate_screenshots(ttl:)
```

Comment:
Moved it up from the `TestHelper::ScreenshotHelper` to make API more readable (since it's supposed to be injected into users code if we go with the first option).